pkg/hasher/hasher.go

```diff
@@ -8,14 +8,27 @@ import (
 )
 
 // HashPassword returns sha512 hash string for provided password
-func HashPassword(password string) string {
+func HashPassword(password string) (string, error) {
+	var passwordHash string
 	hash := sha512.New()
-	hash.Write([]byte(password))
+	_, err := hash.Write([]byte(password))
```

Comment:
I'd suggest use short error check syntax in this way 
if _, err := hash.Write([]byte(password)); err != nil {
    return "", err
}